index.html

```diff
@@ -47,6 +47,148 @@ <h2 class="secondary-text">I'am a Software Developer</h2>
+          <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard</p>
+          <ul class="other-content-tags">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <a class="other-content-link" href="#" target="_blank">See Project</a>
+        </div>
+        <div class="content-defaults content4">
+          <h2 class="other-content-title">Profesional Art Printing Data</h2>
+          <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard</p>
+          <ul class="other-content-tags">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <a class="other-content-link" href="#" target="_blank">See Project</a>
+        </div>
+        <div class="content-defaults content5">
+          <h2 class="other-content-title">Profesional Art Printing Data</h2>
+          <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard</p>
+          <ul class="other-content-tags">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <a class="other-content-link" href="#" target="_blank">See Project</a>
+        </div>
+        <div class="content-defaults content6">
+          <h2 class="other-content-title">Profesional Art Printing Data</h2>
+          <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard</p>
+          <ul class="other-content-tags">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <a class="other-content-link" href="#" target="_blank">See Project</a>
+        </div>
+      </div>
+    </div>
+    <div class="about">
+      <div class="about-top">
+        <h1 class="about-primary-text">About me</h1>
+        <p class="about-description">Hello I’m a software developer! I can help you build a product , feature or website
+          Look through some of my work and experience! If you like what you see and have a project you need coded, don’t
+          hestiate to contact me.</p>
+        <a class="about-link" href="#" target="_blank">Get My Resume</a>
+      </div>
+      <div class="line2"><img src="Images\line2.png" alt="aa"></div>
+
+      <hr class="about-hr">
+      <div class="about-bottom">
+        <ul>
+          <li class="about-bottom-subtopic subtopic-language">Language</li>
+          <div class="about-bottom-items">
+            <ul>
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
```

Comment:
I suggest you remove this from the works section to its own individual section.since this should be a third section of your page.
your page should have three sections
`<body>`
`<header>...</header>`
`<section>headline</section>`
`<section>Works</section>`
`<section>About</section>`
`</body>`

<img width="1659" alt="Screenshot 2022-10-29 at 18 16 29" src="https://user-images.githubusercontent.com/48546992/198840317-87d99946-b29a-4ead-92a7-9eb4b738571b.png">